index.bs

```diff
@@ -204,14 +204,18 @@ NOTE: The dimensions indicated in the 'ispe' property might not match the values
 
 NOTE: The values of [=render_width_minus1=] and [=render_height_minus1=] possibly present in the AV1 bistream are not exposed in the AVIF container level.
 
+<h4 id="clean-aperture-property">Clean Aperture Property</h4>
+
+The semantics of the clean aperture property ('clap') as defined in [[!HEIF]] apply. In addition to the restrictions on transformative item property ordering specified in [[!MIAF]], the following restriction also applies:
+
+<assert>The origin of the 'clap' item property shall be anchored to 0,0 (top-left) unless the full, un-cropped image item is included as a secondary item in an image collection.</assert>
```

Comment:
```diff
The origin of the 'clap' item property shall be anchored to 0,0 (top-left) unless
the full, un-cropped image item is included as a secondary item in an image collection
+containing the item associated with that 'clap' property
```
?

Reply:
What is the point of the image collection if the item is non-hidden?

Reply:
> What is the point of the image collection if the item is non-hidden?

Implementation specific. We simply want to make sure that if a non-zero origin `clap` is used, the full image has to also be viewable. As for why someone would want to create a file like that, it's up to the exact use-case. We had experimental features that played around with ideas like this. (HEIF is currently a bit vague on how an image collection that doesn't contain something like a `ster` or `altr` group is to be interpreted.)

Reply:
Thank you for the explanation. My question was rather:  
If the item is non-hidden, it is viewable (right?). So there is no need to mention an image collection.

Reply:
> So there is no need to mention an image collection.

Ah. I see your point. I guess we should be covered if we say that it needs to be a secondary "**non-hidden** image item" since image item means it's an image collection.

What I guess I'm trying to say is that I don't think it should be enough that the non-cropped image is part of an image sequence since that frame might have such a short duration that it's not really visible.